### PR TITLE
add variables

### PR DESCRIPTION
**Pull request by jpfeuffer**
## Describe the change
Allows fixing #91 

## PR checklist

- [ ] I have added description of the change I'm proposing in the OpenMS Documentation.
- [ ] I have read and followed [OpenMS Documentation Contributing guidelines](CONTRIBUTING.md).
- [ ] I have attached a screenshot of the relevant area after this change.
- [ ] `CHANGELOG.md` is updated.
- [ ] I have added my name in [CONTRIBUTING.md](CONTRIBUTING.md#openms-documentation-contributors).

### CODE REVIEW

File: conf.py [hunk 1]
```diff
@@ -21,6 +21,19 @@
 copyright = '2022, OpenMS Team'
 author = 'OpenMS Team'
 
+
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+#
+# The short X.Y version.
```
Comment (timosachsenberg) [inline conf.py]:
```suggestion
# The short major.minor.patch version.
```

File: conf.py [hunk 2]
```diff
@@ -101,3 +105,12 @@
...
 
 root_doc = 'docs/index'
+
+variables_to_export = [
+    "project",
+    "version",
+    "knime_version"
+]
+frozen_locals = dict(locals())
+rst_epilog = '\n'.join(map(lambda x: f".. |{x}| replace:: {frozen_locals[x]}", variables_to_export))
```
Comment (timosachsenberg) [inline conf.py]: can you add a comment what this does?

**Review by timosachsenberg — APPROVED** [review]

**Review by tapaswenipathak — COMMENTED** [review]
1. did you test if this substitution works in `.rst`,`markdown`, `myst`, or in a config file formatting? or changelog?

2. can you access OpenMS docs `version` in some other repository? say OpenMS/OpenMS?

File: conf.py (hunk 2 above)
Comment (jpfeuffer) [inline conf.py]:
```suggestion
# Makes variables_to_export available in the epilog
rst_epilog = '\n'.join(map(lambda x: f".. |{x}| replace:: {frozen_locals[x]}", variables_to_export))
```

**Review by timosachsenberg — APPROVED** [review]

**Comment by timosachsenberg** [comment]
After conflict resolved

**Comment by jpfeuffer** [comment]
> 1. did you test if this substitution works in `.rst`,`markdown`, `myst`, or in a config file formatting? or changelog?

Yes, see my example.

> 2. can you access OpenMS docs `version` in some other repository? say OpenMS/OpenMS?

No. That is the disadvantage of separate repositories and my initial concern of not being able to "sync" versions. But that is how it is now.

**Comment by jpfeuffer** [comment]
By the way, I kind of reverted the version to 2.7.0 because 2.8.0 was not an official, untested release for conda only.

**Pull request merged**